Examples/Framework/include/ActsExamples/EventData/SimSpacePoint.hpp

```diff
@@ -113,6 +113,21 @@ class SimSpacePoint {
     return m_validDoubleMeasurementDetails;
   }
 
+  // FTF seeding needs function to say if SP is a pixel 
+  // says pixels have 1 SL 
+  bool isPixel() const {
+    if (m_sourceLinks.size() == 1) {
+      return true ;
+    }
+    else {
+      return false ; 
+    }
+  }
+
+  Scalar phi() const {
+    return std::atan(m_y/m_x)  ; 
+
+  }
```

Comment:
Can we find another way to do the same thing? It would be good not to have to change the generic SP definition, particularly introducing concepts like `isPixel` that are ATLAS-specific.